outputs.tf

```diff
@@ -1,35 +1,35 @@
 output "id" {
-  description = "The id of the SCC instance created by this module"
-  value       = resource.ibm_resource_instance.scc_instance.id
+  description = "The id of the SCC instance."
+  value       = local.scc_instance_crn
   # Don't return the SCC ID until it has been configred with COS, as it can't be used until COS integration complete
   depends_on = [ibm_scc_instance_settings.scc_instance_settings]
 }
 
 output "guid" {
-  description = "The GUID of the SCC instance created by this module"
-  value       = resource.ibm_resource_instance.scc_instance.guid
-  # Don't return the SCC GUI until it has been configred with COS, as it can't be used until COS integration complete
+  description = "The GUID of the SCC instance."
+  value       = local.scc_instance_guid
+  # Don't return the SCC GUID until it has been configred with COS, as it can't be used until COS integration complete
   depends_on = [ibm_scc_instance_settings.scc_instance_settings]
 }
 
 output "crn" {
-  description = "The CRN of the SCC instance created by this module"
-  value       = resource.ibm_resource_instance.scc_instance.crn
-  # Don't return the SCC GUI until it has been configred with COS, as it can't be used until COS integration complete
+  description = "The CRN of the SCC instance."
+  value       = local.scc_instance_crn
+  # Don't return the SCC CRN until it has been configred with COS, as it can't be used until COS integration complete
   depends_on = [ibm_scc_instance_settings.scc_instance_settings]
 }
 
 output "name" {
-  description = "The name of the SCC instance created by this module"
-  value       = resource.ibm_resource_instance.scc_instance.name
+  description = "The name of the SCC instance."
+  value       = var.existing_scc_instance_crn == null ? resource.ibm_resource_instance.scc_instance[0].name : data.ibm_resource_instance.scc_instance[0].name
```

Comment:
I don't see name here? https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/data-sources/resource_instance#attribute-reference

Reply:
<img width="462" alt="Screenshot 2024-07-04 at 8 31 54 PM" src="https://github.com/terraform-ibm-modules/terraform-ibm-scc/assets/33781996/a7304b36-3f35-43a6-a427-82e3803b1398">

Name is available in data look up might be a documentation error